packages/fast-components-react-msft/src/select/select.schema.ts

```diff
@@ -103,6 +103,11 @@ export default {
                     title: "Fixed after initial placement",
                     type: "boolean",
                 },
+                scaleToFit: {
+                    title: "Scale to fit",
+                    type: "boolean",
+                    default: "false",
```

Comment:
also here, this should be without quotes or it will not be considered a boolean, it will be a string.